IMAGES.rst

```diff
@@ -0,0 +1,304 @@
+* ARG AIRFLOW_HOME=/opt/airflow - Airflow's HOME (that's where logs and sqlite databases are stored)
+* ARG AIRFLOW_UID="50000" - Airflow user UID
+* ARG AIRFLOW_GID="50000" - Airflow group GID
+* ARG PIP_VERSION="19.0.2" - version of PIP to use
+* ARG CASS_DRIVER_BUILD_CONCURRENCY="8" - Number of processors to use for cassandra PIP install (speeds up
+       installing in case cassandra extra is used).
+
+Those variables should be overwritten only if the production image is built from local sources
+rather than from GitHub repository.
+
+* COPY_SOURCE - should be set to "." if you the image is built from sources
+* COPY_TARGET - should be set to a airflow source directory inside container if the image is built
+  from sources
+* AIRFLOW_SOURCES - should point to the same as COPY_TARGET in case production image is installed from
+  the local sources rather than from Github repository. If left empty it points to sources from
+  GitHub repository derived from AIRFLOW_ORG, AIRFLOW_REPO, AIRFLOW_GIT_REFERENCE
+* CONSTRAINT_REQUIREMENTS should point to locally available requirements file if the image is built
+  from sources. If left empty it will point to the right requirement file from GitHub repository
+  derived from AIRFLOW_ORG, AIRFLOW_REPO, REQUIREMENTS_GIT_REFERENCE
+* ENTRYPOINT_FILE - sources point to entrypoint.sh file when installing from sources if left empty
+  it points to sources from GitHub repository derived from
+  AIRFLOW_ORG, AIRFLOW_REPO, REQUIREMENTS_GIT_REFERENCE
+
+The below will production image in version 3.6 with default extras from master branch in Github:
+
+.. code-block::
+
+  docker build .
+
+The below will build the production image in version 3.7 with default extras from 1.10.9 tag and
+requirements taken from v1-10-test branch in Github. Note that versions 1.10.9 and below
+have no requirements so requirements should be taken from head of the v1-10-test branch. Once we
+release 1.10.10 we can take them from the 1.10.10 tag.
+
+.. code-block::
+
+  docker build . --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7 --build-arg AIRFLOW_GIT_REFERENCE=1.10.9 \
+    --build-arg REQUIREMENTS_GIT_REFERENCE=v1-10-test --build-arg WWW_FOLDER="www_rbac"
+
+The below will build the production image in version 3.7 with default extras from master branch in Github.
+
+.. code-block::
+
+  docker build . --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7
+
+The below will build the production image in version 3.6 with default extras from current sources.
+
+.. code-block::
+
+  docker build . --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7 --build-arg COPY_SOURCE=. \
+    --build-arg COPY_TARGET=/opt/airflow --build-arg AIRFLOW_SOURCES=/opt/airflow \
+    --build-arg CONSTRAINT_REQUIREMENTS=requirements/requirements-python3.7.txt" \
```

Comment:
A starting double-quote fell out. I appreciate your work. Thank you.

Reply:
Thanks!